src/lib/RelayOrderLib.sol

```diff
@@ -4,39 +4,42 @@ pragma solidity ^0.8.0;
 import {IPermit2} from "permit2/src/interfaces/IPermit2.sol";
 import {ISignatureTransfer} from "permit2/src/interfaces/ISignatureTransfer.sol";
 import {PermitHash} from "permit2/src/libraries/PermitHash.sol";
-import {RelayOrder, Input, ResolvedInput} from "../base/ReactorStructs.sol";
+import {RelayOrder, Input, FeeEscalator} from "../base/ReactorStructs.sol";
 import {ReactorErrors} from "../base/ReactorErrors.sol";
-import {InputsLib} from "./InputsLib.sol";
+import {FeeEscalatorLib} from "./FeeEscalatorLib.sol";
 
 library RelayOrderLib {
     using RelayOrderLib for RelayOrder;
-    using InputsLib for Input[];
-    using InputsLib for ResolvedInput[];
+    using FeeEscalatorLib for FeeEscalator;
 
     string internal constant PERMIT2_ORDER_TYPE = string(
-        abi.encodePacked("RelayOrder witness)", RELAY_ORDER_TYPESTRING, PermitHash._TOKEN_PERMISSIONS_TYPESTRING)
+        abi.encodePacked(
+            "RelayOrder witness)",
+            FeeEscalatorLib.FEE_ESCALATOR_TYPESTRING,
+            RELAY_ORDER_TYPESTRING,
+            PermitHash._TOKEN_PERMISSIONS_TYPESTRING
+        )
     );
 
-    /// @dev Max amounts and token addresses are signed in the token permissions of the permit information.
+    /// @dev input token addresses are signed in the token permissions of the permit information.
     bytes internal constant RELAY_ORDER_TYPESTRING = abi.encodePacked(
         "RelayOrder(",
         "address reactor,",
         "address swapper,",
-        "uint256[] startAmounts,",
-        "address[] recipients,",
-        "uint256 decayStartTime,",
-        "uint256 decayEndTime,",
-        "bytes[] actions)"
+        "address inputRecipient,",
+        "FeeEscalator fee,",
+        "bytes data)"
```

Comment:
can we still call this actions? data is vague. or commands?

Reply:
renamed to actions